runtime/doc/options.txt

```diff
@@ -5944,7 +5944,7 @@ A jump table for the options with a short description can be found at |Q_op|.
 	an expensive expression can negatively affect render performance.
 
 					*'statusline'* *'stl'* *E540* *E542*
-'statusline' 'stl'	string	(default "")
+'statusline' 'stl'	string	(default "%<%f %h%w%{%&modified ? '[+] ' : ''%}%r%=%{% &showcmdloc == 'statusline' ? '%-10.S ' : '' %}%{% exists('b:keymap_name') ? '<'..b:keymap_name..'> ' : '' %}%{% &ruler ? ( &rulerformat == '' ? '%-14.(%l,%c%V%) %P' : &rulerformat ) : '' %}")
```

Comment:
This doc is autogenerated. However, this default value far exceeds reasonable linewidth. What's the way to control this?

Reply:
don't need to worry about this.